tasks/publish.py

```diff
@@ -23,21 +17,14 @@ def publish(ctx, base_url, site_prefix, bucket, cache_control,
     '''
     Publish the built site to S3.
 
-    Expects AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY to be
-    in the environment.
+    Requires environment variables: AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY
     '''
-    LOGGER.info('Publishing to S3')
 
-    access_key_id = os.environ['AWS_ACCESS_KEY_ID']
-    secret_access_key = os.environ['AWS_SECRET_ACCESS_KEY']
+    print('Publishing to S3')
 
     start_time = datetime.now()
 
-    s3_client = boto3.client(
-        service_name='s3',
-        region_name=aws_region,
-        aws_access_key_id=access_key_id,
-        aws_secret_access_key=secret_access_key)
+    s3_client = boto3.client(service_name='s3', region_name=aws_region)
```

Comment:
is it the case that the access_key_id and aws_secret_access_key no longer needed if the env variables exist?

Reply:
yeah, they can either be in the environment, provided as arguments, or one of the other million ways to specify them...

https://boto3.amazonaws.com/v1/documentation/api/latest/guide/configuration.html#guide-configuration